src/components/CustomModal/customModal.module.scss

```diff
@@ -0,0 +1,10 @@
+.modalListSection {
+    max-height: 33vh;
+    overflow-y: auto;
+}
+.modaButton {
```

Comment:
Is this `.modalButton` or `.modaButton`. Type maybe?

Reply:
updated the name